src/components/Todos/Todos.tsx

```diff
@@ -0,0 +1,129 @@
+import React from 'react';
+
+import { Todo } from '../../types/Todo';
+
+type Props = {
+  todos: Todo[]
+  onRemove: (todoId: number) => void
+};
+
+export const Todos: React.FC<Props> = ({
+  todos,
+  onRemove,
+}) => {
+  return (
+    <section className="todoapp__main">
+      {todos.map(todo => (
+        <div key={todo.id} className={`todo ${todo.completed && 'completed'}`}>
+          <label className="todo__status-label">
+            <input
+              type="checkbox"
+              className="todo__status"
+            />
+          </label>
+
+          <span className="todo__title">{todo.title}</span>
+
+          {/* Remove button appears only on hover */}
+          <button
+            type="button"
+            className="todo__remove"
+            onClick={() => onRemove(todo.id)}
+          >
+            ×
+          </button>
+
+          {/* overlay will cover the todo while it is being updated */}
+          <div className="modal overlay">
+            <div className="modal-background has-background-white-ter" />
+            <div className="loader" />
+          </div>
+        </div>
+      ))}
+    </section>
+
+  // <section className="todoapp__main">
```

Comment:
remove comments (you always can find it in `master` branch if you need it in the next tasks)

Reply:
Not fixed